flow-model-generator/modules/flow-model-generator-ls-extension/src/test/resources/module_nodes/config/agent_model.json

```diff
@@ -29,14 +29,14 @@
               "offset": 2
             }
           },
-          "sourceCode": "final ai:Agent agent = check new (\n    systemPrompt = {\n        role: \"Telegram Assistant\",\n        instructions: \"Assist the users with their requests, whether it's for information, \" +\n            \"tasks, or troubleshooting. Provide clear, helpful responses in a friendly and professional manner.\"\n    },\n    model = model,\n    tools = [sum, multiply],\n    memory = new ai:MessageWindowChatMemory(10) // Available by default\n);"
+          "sourceCode": "final ai:Agent agent = check new (\n    systemPrompt = {\n        role: \"Telegram Assistant\",\n        instructions: \"Assist the users with their requests, whether it's for information, \" +\n            \"tasks, or troubleshooting. Provide clear, helpful responses in a friendly and professional manner.\"\n    },\n    model = model,\n    tools = [sum, multiply],\n    memoryManager = new ai:DefaultMessageWindowChatMemoryManager(10) // Available by default\n);"
```

Comment:
@MohamedSabthar, you have changed the source file of this test to `memoryManager = new ai:DefaultMessageWindowChatMemoryManager` but the existing one `memory = new ai:MessageWindowChatMemory(10)` is correct. Any reason for this?

Reply:
We can revert to the correct version. I believe the API was changed after a few iterations.